app/Repositories/Summit/DoctrinePresentationActionTypeRepository.php

```diff
@@ -29,30 +33,41 @@ final class DoctrinePresentationActionTypeRepository
     /**
      * @inheritDoc
      */
-    protected function getBaseEntity()
+    protected function getBaseEntity(): string
     {
        return PresentationActionType::class;
     }
 
+    /**
+     * @param QueryBuilder $query
+     * @param Filter|null $filter
+     * @return QueryBuilder
+     */
+    protected function applyExtraJoins(QueryBuilder $query, ?Filter $filter = null): QueryBuilder
+    {
+        return $query->leftJoin("e.assigned_selection_plans", 'sp', Join::ON);
```

Comment:
this is not correct
u should be doing

$query->leftJoin("e.assigned_selection_plans", 'asp')->lefJoind('asp.selection_plan", "sp");

Reply:
done